package/gluon-core/check_site.lua

```diff
@@ -28,6 +28,15 @@ for _, config in ipairs({'wifi24', 'wifi5'}) do
     need_string('regdom') -- regdom is only required when wifi24 or wifi5 is configured
 
     need_number(config .. '.channel')
+
+    local rates={1000, 2000, 5500, 6000, 9000, 11000, 12000, 18000, 24000, 36000, 48000, 54000}
+    local supported_rates =  need_array(config .. '.supported_rates', nil, false)
+    if supported_rates then
+      need_array_of(config .. '.supported_rates', rates, true)
```

Comment:
It doesn't make sense to check supported_rates twice. Just do `local supported_rates = need_array_of(config .. '.supported_rates', rates, false)` above, and only check basic_rate in the if/else branches.